### PR TITLE
Update flying bot descriptions

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
```SUMMARY: Content "Minor updates to descriptive text on flying bots"```

#### Purpose of change
I've heard a bit of confusion about whether or not antigravity exists in game, given the existence of flying robots. The descriptive text on these bots was pretty unclear.

#### Describe the solution
Clarify that drones are probably all quadcopters

#### Describe alternatives you've considered
Other UAV options would work, but quadcopters make a lot of sense for these applications.

#### Additional context
I also gave a bit more information about the eyebot just for fun.

### CODE REVIEW

**Comment by RarkGrames** [comment]
The base manhacks could use a little more description in my opinion. Until the change it was pretty clear that they are the little drones from Half-life 2. Obviously those drones wouldn't work in reality, but now I have no idea how do they look like or how do they attack with their blades (rotors?).

**Comment by I-am-Erk** [comment]
Could easily just be a quadcopter with blades for propellers, which is what I assumed. Maybe it's got a little spinning extra blade somewhere.

File: data/json/monsters.json [hunk 1]
```diff
@@ -1033,7 +1033,7 @@
     "id": "mon_eyebot",
     "type": "MONSTER",
     "name": "eyebot",
-    "description": "A fusion-driven UAV largely comprised of a high-resolution camera lens and a speaker, this spheroid robot hovers above the ground, documenting the carnage and mayhem around it.",
+    "description": "A fusion-driven quadcopter drone largely comprised of a high-resolution camera lens and a speaker, this robot hovers above the ground, documenting the carnage and mayhem around it.  While it is no longer able to report to a central authority, it's likely that some of its automated alert systems are still functional, given that the thing seems to have an operational charging station somewhere.",
```
Comment (kevingranade) [inline data/json/monsters.json]:
```suggestion
    "description": "A quadcopter drone largely comprised of a high-resolution camera lens and a speaker, this robot hovers above the ground, documenting the carnage and mayhem around it.  While it is no longer able to report to a central authority, it's likely that some of its automated alert systems are still functional, given that the thing seems to have an operational charging station somewhere.",
```

Comment (Throwaway-name) [reply data/json/monsters.json]: @kevingranade you overwrote the quadcopter part.

**Pull request merged**